### ISSUE TITLE
FSR Disable on standalone mod and Potato Patch mod differ

### ISSUE THREAD

**Issue by iAbadia** [HolographicWings/TOTK-Mods-collection#5]
From Disable FSR Mod:
```
// Disable FSR downscaling
@enabled
00CAD340 08008052
@stop
```

From Potato Patch:
```
// patchanon Disable FSR downscaling
@enabled
00CAD348 08008052
@stop
```

Notice last byte on the address. Some users are reporting issues with the standalone mod, suggesting that the address on Potato is right.

**Comment by iAbadia** [comment]
Addressed on PR#6

https://github.com/HolographicWings/TOTK-Mods-collection/pull/6

**Comment by HolographicWings** [comment]
> Addressed on PR#6
> 
> #6

Yes, according to Julienraptor01, both of them are wrong and the patch only has a chance to take effect.
The good value being 00CAD344, PR being merged

**Comment by Julienraptor01** [comment]
Imagine both are wrong, just imagine...
